web/portal/src/main/webapp/onboarding/jsp/reset_password.jsp

```diff
@@ -92,24 +92,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title><%=res.getString("onboarding.changePass.title")%></title>
     <%if (success != null && !success.isEmpty()) {%>
-        <meta http-equiv="refresh" content="5; url=<%=contextPath+ "/login"%>" />
+        <meta http-equiv="refresh" content="1; url=<%=contextPath+ "/login"%>" />
     <%}%>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
     <link rel="shortcut icon" type="image/x-icon"  href="<%=contextPath%>/favicon.ico" />
     <link id="brandingSkin" rel="stylesheet" type="text/css" href="/rest/v1/platform/branding/css">
     <link href="<%=loginCssPath%>" rel="stylesheet" type="text/css"/>
-	<link rel="stylesheet" type="text/css" href="<%=contextPath%>/login/skin/Stylesheet.css"/>
+	<link rel="stylesheet" type="text/css" href="<%=contextPath%>/login/skin/Onboarding.css"/>
```

Comment:
- Remove css classes specific to onboarding from Stylesheet.css
- Keep Stylesheet.css link import for common css classes between onboarding and login
- Keep onboarding.css for specific onboarding css classes and overwritten login css classes